src/_gettsim/interface.py

```diff
@@ -513,7 +513,7 @@ def _fail_if_data_tree_not_valid(data_tree: NestedDataDict) -> None:
     """
     assert_valid_gettsim_pytree(
         tree=data_tree,
-        leaf_checker=lambda leaf: isinstance(leaf, pd.Series | np.ndarray),
+        leaf_checker=lambda leaf: isinstance(leaf, pd.Series | np.ndarray | list),
```

Comment:
We should rather adjust the loader as opposed to being less strict just for the tests, I think. Should return a dict of arrays, see below.